utils/inference_utils.py

```diff
@@ -846,7 +846,16 @@ def run_inference(examples, serving_bundle):
   elif serving_bundle.custom_predict_fn:
     # If custom_predict_fn is provided, pass examples directly for local
     # inference.
-    values = serving_bundle.custom_predict_fn(examples)
+    from inspect import signature
```

Comment:
It's better to pull this import to module level as custom_predict_fn is supported in all environments now (TB, notebook etc.).